stories/Components/Map/Map.mdx

```diff
@@ -56,16 +57,13 @@ import Fetcher from '../../Fetcher/Fetcher';
 const MapComponentWithFetcher = () => (
   <Fetcher
     api="https://sfvc-migration.undrr.org/api/v2/content/commitments-metrics"
-    queryParams={{items_per_page:1000}}
     render={({ isLoading, data }) => (
       <>
         {isLoading ? (
-          <span>Loading data.</span>
+          <Loader />
         ) : (
           <MapComponent
-            data={transformData(data)}
```

Comment:
We've lost the `trasnformData` wrapper function, is that intentional?

Reply:
I accidentally removed it, i am bringing it back, sorry